docker-compose.yml

```diff
@@ -3,14 +3,18 @@ version: '3.8'
 x-app: &app
   build:
     context: .
-    target: hyku-base
+    target: hyku-web
     args:
       - SETTINGS__BULKRAX__ENABLED=true
+    cache_from:
+      - ghcr.io/scientist-softserv/adventist-dl/base:latest
+      - ghcr.io/scientist-softserv/adventist-dl/web:latest
```

Comment:
i'm rebuilding with the changes and I keep seeing this

<img width="897" alt="image" src="https://user-images.githubusercontent.com/19597776/216394071-6807a301-d77a-4a30-b040-9d7cdd13d713.png">

could there be a naming issue?

Reply:
Those are the same lines we tried adding to BL and removed again for the same reason. 

Does it build with them as is in spite of the errors? Wondering if we should remove them, or leave but note as something to be addressed.

Reply:
@laritakr yup the app still spins up just fine